ui/imodel-components-react/src/test/inputs/QuantityNumberInput.test.tsx

```diff
@@ -472,13 +472,13 @@ describe("<QuantityNumberInput />", () => {
       const wrapper = render(<QuantityNumberInput persistenceValue={value} quantityType={QuantityType.Length} step={.25} onChange={handleChange} />);
       const input = wrapper.container.querySelector("input");
       expect(input).not.to.be.null;
-      expect((input as HTMLInputElement).value).to.eq("1.");
+      expect((input as HTMLInputElement).value).to.eq("1.0");
```

Comment:
@pmconne mentioned that this file will be moved to iTwin/appuii repo (https://github.com/iTwin/itwinjs-core/pull/5149).
I've had a word with Arun about deleting the file when it conflicts with his PR.
@NancyMcCallB @raplemie this will cause test failures in your repos.